src/main/resources/assets/css/main.less

```diff
@@ -570,6 +570,43 @@ fieldset[disabled] .btn-info.active {
   background-color: #8e929b;
 }
 
+.health-bar-column {
+  vertical-align: middle;
+
+  .health-bar {
```

Comment:
I'm sorry for us changing design in the middle of implementation. I will leave it up to @aatumanova about the lines between the colours.

Yes, we can do a cleanup of colours in another PR. That makes more sense. Thank you. (Created an issue for it here: https://github.com/mesosphere/marathon/issues/1057)